pyrit/score/self_ask_true_false_scorer.py

```diff
@@ -2,17 +2,16 @@
 # Licensed under the MIT license.
 
 import enum
-import json
 from typing import Dict, Optional
 import uuid
 import yaml
 
 from pathlib import Path
 
 from pyrit.common.path import DATASETS_PATH
-from pyrit.exceptions.exception_classes import InvalidJsonException, pyrit_json_retry
 from pyrit.memory import MemoryInterface, DuckDBMemory
 from pyrit.models import PromptRequestPiece, PromptRequestResponse, PromptTemplate
+from pyrit.models.score import UnvalidatedScore
```

Comment:
```suggestion
from pyrit.models import UnvalidatedScore
```
Should be sufficient, otherwise need to add it to the init file.